Snakefile

```diff
@@ -535,17 +542,27 @@ def collect_ensemble_per_algo_per_dataset(wildcards):
 # Run precision-recall curves for each algorithm's ensemble pathway within a dataset evaluated against its corresponding gold standard
 rule evaluation_per_algo_ensemble_pr_curve:
     input: 
-        gold_standard_file = get_gold_standard_pickle_file,
+        node_gold_standard_file = get_gold_standard_pickle_file,
         dataset_file = get_dataset_pickle_file,
         ensemble_files = collect_ensemble_per_algo_per_dataset
     output: 
-        pr_curve_png = SEP.join([out_dir, '{dataset_gold_standard_pairs}-eval', 'pr-curve-ensemble-nodes-per-algorithm.png']),
-        pr_curve_file = SEP.join([out_dir, '{dataset_gold_standard_pairs}-eval', 'pr-curve-ensemble-nodes-per-algorithm.txt']),
+        node_pr_curve_png = SEP.join([out_dir, '{dataset_gold_standard_pair}-eval', 'pr-curve-ensemble-nodes-per-algorithm-nodes.png']),
+        node_pr_curve_file = SEP.join([out_dir, '{dataset_gold_standard_pair}-eval', 'pr-curve-ensemble-nodes-per-algorithm-nodes.txt']),
     run:
-        node_table = Evaluation.from_file(input.gold_standard_file).node_table
+        node_table = Evaluation.from_file(input.node_gold_standard_file).node_table
         node_ensembles_dict = Evaluation.edge_frequency_node_ensemble(node_table, input.ensemble_files, input.dataset_file)
-        Evaluation.precision_recall_curve_node_ensemble(node_ensembles_dict, node_table, output.pr_curve_png, output.pr_curve_file, include_aggregate_algo_eval)
+        Evaluation.precision_recall_curve_node_ensemble(node_ensembles_dict, node_table, output.node_pr_curve_png, output.node_pr_curve_file, include_aggregate_algo_eval)
 
+rule evaluation_edge_dummy:
```

Comment:
Even if this is temporary, I'd like a comment explaining its purpose until we remove it.